controllers/MobileAppController.php

```diff
@@ -6,11 +6,18 @@
 use humhub\modules\admin\components\Controller;
 use humhub\modules\admin\notifications\NewVersionAvailable;
 use humhub\modules\fcmPush\models\FcmUser;
+use humhub\modules\fcmPush\Module;
 use humhub\modules\user\models\User;
 use Yii;
+use yii\web\Response;
 
 class MobileAppController extends Controller
```

Comment:
@yurabakhtin Currently this controller is only avaialble for admins.

I would suggest to move the other method into another controller.

Reply:
@luke- Fixed in the commit https://github.com/humhub/fcm-push/pull/38/commits.